bin/publish-github-release/index.js

```diff
@@ -62,13 +63,16 @@ function formGithubReleaseResource({ version, title, description, branch }) {
   if (description) {
     releaseResource.body = description;
   }
+  if (preRelease !== undefined) {
```

Comment:
Why wouldn't a falsy check suffice?

Reply:
i wanted to be able to pass "false" if you wanted to. Doesn't make a difference though.